pygmt/tests/baseline/test_solar_default_terminator.png.dvc

```diff
@@ -0,0 +1,4 @@
+outs:
```

Comment:
This dvc file is no longer needed and should be removed from git.

Reply:
done in  2490763